packages/cloud-cognitive/src/components/CreateTearsheet/CreateTearsheet.js

```diff
@@ -64,11 +70,33 @@ export let CreateTearsheet = forwardRef(
     const [onNext, setOnNext] = useState();
     const [onMount, setOnMount] = useState();
     const [stepData, setStepData] = useState([]);
-    const totalStepCount = React.Children.count(children);
+    const [firstIncludedStep, setFirstIncludedStep] = useState(1);
+    const [lastIncludedStep, setLastIncludedStep] = useState(null);
 
     const previousState = usePreviousValue({ currentStep, open });
     const contentRef = useRef();
 
+    const lastIndexOf = (array, key) => {
+      for (let i = array.length - 1; i >= 0; i--) {
+        if (array[i]?.[key] === true) {
+          return i + 1;
+        }
+      }
+      return -1;
+    };
+
+    useEffect(() => {
```

Comment:
if this is the same code used in `CreateFullPage` you should definitely make them a shared utility

Reply:
Created utility called `lastIndexOf.js`